auth/store.go

```diff
@@ -114,6 +112,9 @@ type AuthStore interface {
 
 	// IsAdminPermitted checks admin permission of the user
 	IsAdminPermitted(username string) bool
+
+	// GenSimpleToken produces a simple random string
+	GenSimpleToken() (string, error)
```

Comment:
do we really need to make gen simple token as an interface of auth? even if we need, probably we should make this a gentoken? (we will have jtw token)